resources/krasa/grepconsole/icons/highlight.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I didn't have to do this, I could just have my themes just replace these icons. It felt better just to update these to [action icons](https://jetbrains.design/intellij/principles/icons/#action-icons), that way they follow the LaF provided by any theme. 

I can remove these changes, if wanted.